plugin/hooks.go

```diff
@@ -36,6 +36,11 @@ const (
 	TotalHooksId            = iota
 )
 
+const (
+	//DismissPostError dismisses a pending post when the error is returned from MessageWillBePosted.
```

Comment:
Nit: There should be a whitespace.
```suggestion
	// DismissPostError dismisses a pending post when the error is returned from MessageWillBePosted.
```